### PR TITLE
[Add] add api version in response json

### PR DESCRIPTION
**Pull request by wu5bocheng**
add dynamic version to the responses

### CODE REVIEW

File: Docs/API_design_doc.md [hunk 1]
```diff
@@ -32,11 +32,12 @@
...
-        | Name | Type | Description |
-        |----- |------|-------------|
-        | status | int | Status code |
-        | error | JSON | Error information |
-        | data | JSON (list) | [Recommended data](#recommended_data) |
+      | Name | Type | Description                                              |
+      |----- |----------------------------------------------------------|-------------|
+      | status | int | Status code                                              |
+      | error | JSON | Error information                                        |
+      | api_version | string | The API version parsed from environment variables |
```
Comment (zhoxing-ms) [inline Docs/API_design_doc.md]: > The API version parsed from environment variables

The source of the version can be obtained from the code logic. We should explain the meaning of the api version in the doc, such as the `MAJOR` version for major upgrades, the `MINOR` version for new feature, and the `PATCH` version for bugfix

File: HISTORY.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+History
+-------
+
+.. :The first position of the version number represents a major update, the second position represents a new version release, and the third position represents a minor adjustment of the model without re-release.
+
+1.0.0 (2023-03-20)
+~~~~~~~~~~~~~~~~~~
+
+* Command recommendation based on frequency of command combinations
```
Comment (zhoxing-ms) [inline HISTORY.rst]:
```suggestion
* Integrate with offline calculation model, Aladdin model, and E2E scenario knowledge base to provide three use cases of solution recommendation, command recommendation and scenario recommendation
```

File: Docs/API_design_doc.md (hunk 1 above)
Comment (wu5bocheng) [reply Docs/API_design_doc.md]: A short version number description has been added, detailed in HISTORY.rst

**Comment by wu5bocheng** [comment]
Already modified, ready for rereview and merge.

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**